styles.css

```diff
@@ -90,6 +90,61 @@ button {
 
 nav {
   display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  position: absolute;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  right: 0;
+  background-color: #fff;
+}
+
+.nav-on {
+  display: flex;
+}
+
+.navtop {
+  height: 115px;
+  padding-right: 20px;
+  padding-top: 25px;
+  background: top left url(images/image_geometry_menu_1.svg) no-repeat;
+  text-align: right;
+}
+
+.navbottom {
+  height: 150px;
+  background: bottom left url(images/image_geometry_menu_2.svg) no-repeat;
+}
+
+.main-nav {
+  flex: calc(100% - 265px);
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.main-nav li {
+  margin: 0;
+  padding: 30px 25px 0 25px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 300%;
+  text-align: center;
+}
```

Comment:
- [ ] Great job styling the implementing the mobile menu :clap: However it would be nice to reduce the margin between the menu option to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-06-20 08-20-29](https://github.com/balatstar/portfolio/assets/91142432/7a9671bf-acf3-4313-8718-23da2133daa2)
### What it's supposed to look like :point_down: 
![Screenshot from 2023-06-20 08-24-55](https://github.com/balatstar/portfolio/assets/91142432/e00d2907-2cad-4c69-9ac2-018a833cd0d3)

  -  **HIT:** `padding: 0 25px`